### PR TITLE
CI: add pypy 3.7

### PR DESCRIPTION
**Pull request by RinCat**

### CODE REVIEW

File: tox.ini [hunk 1]
```diff
@@ -10,6 +10,7 @@ python =
     3.9: py39
     3.10: py310
     pypy-3.6: pypy3
+	pypy-3.7: pypy3
```
Comment (mgorny) [inline tox.ini:13]: Indent?

Comment (RinCat) [reply tox.ini]: Oops, edited in github, it used wrong indent but looks the same...

**Comment by zmedico** [comment]
Today I guess there was some kind of update and pypy-3.6 stopped working, so I replaced it with pypy-3.7-v7.3.3 in 14d2182465f9921717801563f83b376cfa439bd1.

However, tox isn't running the tests for some reason, as seen in this run for example:

https://github.com/gentoo/portage/runs/1949369910?check_suite_focus=true

It only produced this output:
```
Run tox -vv
  tox -vv
  shell: /usr/bin/bash -e {0}
  env:
    pythonLocation: /opt/hostedtoolcache/PyPy/3.7.9/x64/bin
using tox.ini: /home/runner/work/portage/portage/tox.ini (pid 2281)
running tox-gh-actions
original envconfigs: ['py36', 'py37', 'py38', 'py39', 'py310', 'pypy3']
original envlist_default: ['py36', 'py37', 'py38', 'py39', 'py310', 'pypy3']
original envlist: ['py36', 'py37', 'py38', 'py39', 'py310', 'pypy3']
Python versions: ['pypy-3.7', 'pypy-3', 'pypy3']
tox-gh-actions config: {'python': {'3.6': ['py36'], '3.7': ['py37'], '3.8': ['py38'], '3.9': ['py39'], '3.10': ['py310'], 'pypy-3.7-v7.3.3': ['pypy3']}, 'env': {}}
using the following factors to decide envlist: []
overriding envlist with: []
using tox-3.22.0 from /opt/hostedtoolcache/PyPy/3.7.9/x64/site-packages/tox/__init__.py (pid 2281)
skipping sdist step
___________________________________ summary ____________________________________
  congratulations :)
```

**Comment by RinCat** [comment]
Seems tox cannot include the detailed version, but use the CI provided one.

**Comment by RinCat** [comment]
I can reproduce the error on both official binary pypy37 and local pypy37, but can't find out why. 
The `array` module can be imported from shell, and portage test cases passed.

```
PYTHONPATH=$PWD/lib:$PWD/repoman/lib python -m pylint -v *                                                                                                                                                                                                  2 ↵
************* Module bin.xattr-helper
bin/xattr-helper.py:17:0: E0401: Unable to import 'array' (import-error)
************* Module lib.portage.xpak
lib/portage/xpak.py:25:0: E0401: Unable to import 'array' (import-error)
************* Module lib.portage.tests.ebuild.test_array_fromfile_eof
lib/portage/tests/ebuild/test_array_fromfile_eof.py:4:0: E0401: Unable to import 'array' (import-error)
************* Module lib._emerge.AbstractPollTask
lib/_emerge/AbstractPollTask.py:4:0: E0401: Unable to import 'array' (import-error)

------------------------------------
Your code has been rated at 10.00/10
```

**Comment by RinCat** [comment]
This one seems a bug in pylint 2.7.0, I downgrade to 2.6.2 and it passed.

**Comment by thesamesam** [comment]
I think we merged this since.

**Pull request closed without merging**